packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -173,10 +188,13 @@ export function generateGCConfigs(
 		sessionExpiryTimeoutMs,
 		sweepTimeoutMs,
 		inactiveTimeoutMs,
-		throwOnInactiveLoad,
 		persistedGcFeatureMatrix,
 		gcVersionInBaseSnapshot,
 		gcVersionInEffect,
+		gcThrowOnInactiveLoad,
```

Comment:
Why preface all these gc configs with `gc`?

Reply:
Initially, I was returning these directly from garbage collector to container runtime. I did not rename them when I changed them to getters.